universite-gustave-eiffel-arts-numeriques-et-cultures-visuelles.csl

```diff
@@ -0,0 +1,253 @@
+      <label form="short" prefix=" (" suffix=".)"/>
+    </names>
+  </macro>
+  <macro name="titre">
+    <choose>
+      <if type="book" match="any">
+        <text variable="title" text-case="capitalize-first" font-style="italic"/>
+      </if>
+      <else-if type="article article-journal article-magazine article-newspaper" match="any">
+        <group>
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="container-title" font-style="italic" prefix=", "/>
+        </group>
+      </else-if>
+      <else-if type="thesis" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" font-style="italic"/>
+          <text variable="genre" suffix=","/>
+          <text variable="publisher"/>
+        </group>
+      </else-if>
+      <else-if type="chapter" match="any">
+        <group suffix=" ">
+          <group delimiter=" ">
+            <text variable="title" text-case="capitalize-first" quotes="true"/>
+            <choose>
+              <if match="any" variable="original-date">
+                <date form="text" variable="original-date" prefix="(" suffix=")"/>
+              </if>
+            </choose>
+          </group>
+          <text value="dans" prefix=", " suffix=" "/>
+          <names variable="container-author" suffix=","/>
+        </group>
+        <text macro="dir-d-ouvrage"/>
+        <text variable="container-title" text-case="capitalize-first" font-style="italic"/>
+      </else-if>
+      <else>
+        <text variable="title" strip-periods="true" quotes="true"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="trad">
+    <names variable="translator" prefix="trad. ">
+      <name and="text" delimiter-precedes-last="never">
+        <name-part name="family"/>
+      </name>
+    </names>
+  </macro>
+  <macro name="volnum">
+    <choose>
+      <if type="book chapter" match="any">
+        <choose>
+          <if match="all" is-numeric="edition">
+            <number variable="edition" form="ordinal"/>
+            <text term="edition" form="short"/>
+          </if>
+          <else>
+            <text variable="edition" text-case="capitalize-first"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="article article-journal article-magazine" match="any">
+        <choose>
+          <if match="all" is-numeric="issued">
+            <text macro="num"/>
+            <text macro="vol"/>
+          </if>
+          <else>
+            <text macro="vol"/>
+            <text macro="num"/>
+          </else>
+        </choose>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="vol">
+    <choose>
+      <if match="all" is-numeric="volume">
+        <text term="volume" form="short" suffix=". "/>
+        <text variable="volume"/>
```

Comment:
put these 2 in a group and set the "." as a delimiter. (please apply the same for other instances in the style. There is a few)